serializations.md

```diff
@@ -529,6 +529,12 @@ The content of the contract store depends on [the ABI and the VM version](/contr
 ]
 ```
 
+The recipient must be one of the following:
```

Comment:
I understand correctly this is clarification that applies to Fortuna already. Up to you if you would prefer to split it as separate PR against master.